src/span.js

```diff
@@ -106,6 +122,9 @@ export default class Span {
             if (this.getBaggageItem(normalizedKey)) {
                 logs['override'] = 'true';
             }
+            if (truncated) {
+                logs['truncated'] = 'true';
```

Comment:
Each baggage item gets its own set of logs. If we were to dump everything into the same logs, then there would be trouble. I think this current approach is fine.